pkg/vcluster-rancher-extension-ui/components/VClusterClusterCreateItem.vue

```diff
@@ -42,15 +42,22 @@ export default {
   },
 
   methods: {
-    openVClusterModal() {
+    async openVClusterModal() {
       this.showModal = true;
-      this.fetchClusters();
-      this.fetchVersions();
-      this.loadAllProjects();
+      try {
+        await this.loadAllProjects();
+        this.fetchClusters();
+        this.fetchVersions();
```

Comment:
Is it intentional that we don't wait for these? If so, would be good to add a comment why